### ISSUE TITLE
Support multiple versions of docker

### ISSUE THREAD

**Issue by MiLk** [AerisCloud/ansible-role-test#8]
```
404 Client Error: Not Found ("client and server don't have same version (client : 1.18, server: 1.17)")
```

**Comment by christopherobin** [comment]
You will need to upstream that to [docker-py](/docker/docker-py)

**Comment by christopherobin** [comment]
Try the latest version of PR #7

**Comment by MiLk** [comment]
Looks ok.

**Comment by christopherobin** [comment]
Fixed in v0.1.0
